### PR TITLE
Added JUnit test for OCPP 1.6 messages

### LINKED ISSUE #206: 134-[Back] Create unitary tests for OCPP packets
┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1206575137133723/1206712852936554) by [Unito](https://www.unito.io)

### PR DESCRIPTION
**Pull request by tvelu77**
Added JUnit test for OCPP 1.6 messages

### CODE REVIEW

File: src/main/java/fr/uge/chargepointconfiguration/chargepointwebsocket/ocpp/ocpp16/BootNotificationRequest16.java [hunk 1]
```diff
@@ -31,9 +31,31 @@ public record BootNotificationRequest16(String chargePointVendor,
...
+      );
+    }
+    if (chargePointSerialNumber != null && chargePointSerialNumber.length() > 25) {
+      throw new IllegalArgumentException(
+              "Chargepoint serial number cannot be longer than 25 characters."
+      );
+    }
+    if (chargeBoxSerialNumber != null && chargeBoxSerialNumber.length() > 25) {
+      throw new IllegalArgumentException(
+              "Chargepoint serial number cannot be longer than 25 characters."
```
Comment (Idhao) [inline src/main/java/fr/uge/chargepointconfiguration/chargepointwebsocket/ocpp/ocpp16/BootNotificationRequest16.java]:
```suggestion
              "ChargeBox serial number cannot be longer than 25 characters."
```

Comment (tvelu77) [reply src/main/java/fr/uge/chargepointconfiguration/chargepointwebsocket/ocpp/ocpp16/BootNotificationRequest16.java]: Oh shoot, I forgor to change, thanks !

Comment (tvelu77) [reply src/main/java/fr/uge/chargepointconfiguration/chargepointwebsocket/ocpp/ocpp16/BootNotificationRequest16.java]: [ba582fb](https://github.com/LastProject-ESIEE/OCPPConfigurationServer/pull/252/commits/ba582fbc5adbebba344a1a274d04201d41b4aac7)

File: src/test/java/fr/uge/chargepointconfiguration/chargepointwebsocket/ocpp/ocpp16/ChangeConfigurationRequest16Test.java [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+            "key",
+            """
+                    Unfortunately, the value is more than 500 characters long.
+                    Furthermore, it should be not accepted as a value according to
+                    the OCPP 1.6 specification edited in 2019...
+                    Please, restrain yourself from putting a long value into the packet
+                    or the constructor would not be happy with you.
+                    You really do not want to have the constructor angry at you and if it is the case,
+                    well, farewell and good luck my friend, because you'll need luck to survive !
+                    Oh no, the constructor is here... Please have mercy, NOOOOOOOOOOOOOOOOOOOOOO"""
```
Comment (Idhao) [inline src/test/java/fr/uge/chargepointconfiguration/chargepointwebsocket/ocpp/ocpp16/ChangeConfigurationRequest16Test.java]:
```suggestion
                    Oh no, the constructor is here... Please have mercy, NOOOOOOOOOOOOOOOOOOOOOO
            """
```

**Review by Idhao — APPROVED** [review]

**Review by Idhao — APPROVED** [review]

**Pull request merged**